python/cac_tripplanner/destinations/views.py

```diff
@@ -130,3 +133,24 @@ def get(self, request, *args, **kwargs):
             results = results[:limit_int]
         data = serializers.serialize('json', results, fields=output_fields)
         return HttpResponse(data, 'application/json')
+
+class FeedEvents(View):
+    """ API endpoint for the FeedEvent model """
+
+    def get(self, request, *args, **kwargs):
+        """ GET 20 most recent feed events that are published
+
+        TODO: Additional filtering, dynamic limits?
```

Comment:
I spoke with @jbranigan yesterday and he said that these FeedEvents would be added to the map as an additional toggleable marker layer that defaults to on. They are not to be shown in the featured destinations section of the homepage. I didn't ask whether they should show up in the search results.